examples/java/pom.xml

```diff
@@ -195,17 +195,6 @@
         </executions>
       </plugin>
     </plugins>
-
-    <extensions>
-      <!-- Use os-maven-plugin to initialize the "os.detected" properties for Bigtable
-           in beam-sdks-java-io-google-cloud-platform.
-           See: https://cloud.google.com/bigtable/docs/using-maven#encryption -->
-      <extension>
-        <groupId>kr.motd.maven</groupId>
-        <artifactId>os-maven-plugin</artifactId>
-        <version>${os-maven-plugin.version}</version>
```

Comment:
you can also remove `os-maven-plugin.version` property definition from the root pom.